client/test/modify.ts

```diff
@@ -1033,6 +1048,76 @@ test.serial('$increment, $default', async (t) => {
     'does not overwrite if value exists'
   )
 
+  await client.set({
+    $id: 'viHelloYes',
+    obj: {
+      value: {
+        $default: 100,
+        $increment: 10,
+      },
+      rec: {
+        test1: {
+          value: {
+            $default: 10,
+            $increment: 11,
+          },
+        },
+      },
+    },
+  })
+
+  t.deepEqual(
+    await client.get({
+      $id: 'viHelloYes',
+      obj: true,
+    }),
+    {
+      obj: {
+        value: 100,
+        rec: {
+          test1: {
+            value: 10,
+          },
+        },
+      },
+    }
+  )
+
+  await client.set({
+    $id: 'viHelloYes',
+    obj: {
+      value: {
+        $default: 100,
+        $increment: 10,
+      },
+      rec: {
+        test1: {
+          value: {
+            $default: 10,
```

Comment:
setting this 11 might make the test cover more cases. the result should still be 21, right?

Reply:
Sure, can't hurt. Updated. Thx!